packages/env/lib/commands/destroy.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+const dockerCompose = require( 'docker-compose' );
+const util = require( 'util' );
+const fs = require( 'fs' ).promises;
+const path = require( 'path' );
+const inquirer = require( 'inquirer' );
+
+/**
+ * Promisified dependencies
+ */
+const rimraf = util.promisify( require( 'rimraf' ) );
+
+/**
+ * Internal dependencies
+ */
+const stop = require( './stop' );
+const { readConfig } = require( '../../lib/config' );
+
+/**
+ * Destroy the development server.
+ *
+ * @param {Object}  options
+ * @param {Object}  options.spinner A CLI spinner which indicates progress.
+ * @param {boolean} options.debug   True if debug mode is enabled.
+ */
+module.exports = async function destroy( { spinner, debug } ) {
+	const configPath = path.resolve( '.wp-env.json' );
+	const { dockerComposeConfigPath, workDirectoryPath } = await readConfig(
+		configPath
+	);
+
+	// check installed WordPress.
+	const installed = await fs.readdir( workDirectoryPath );
+	if ( ! installed.length ) {
+		spinner.text = `Could not find any files to remove.`;
+		return;
+	}
```

Comment:
```suggestion
	try {
		await fs.readdir( workDirectoryPath );
	} catch {
		spinner.text = 'Could not find any files to remove.';
		return;
	}
```
this throws an error when you try to read the directory and it doesn't exist

Sorry for the constant back and forth! after this, it will probably be good to go ;)